src/test/java/com/patientpal/backend/caregiver/service/CaregiverServiceTest.java

```diff
@@ -0,0 +1,190 @@
+import com.patientpal.backend.common.exception.EntityNotFoundException;
+import com.patientpal.backend.member.domain.Member;
+import com.patientpal.backend.member.repository.MemberRepository;
+import com.patientpal.backend.test.annotation.AutoKoreanDisplayName;
+import java.util.Optional;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+@SuppressWarnings("NonAsciiCharacters")
+@AutoKoreanDisplayName
+class CaregiverServiceTest {
+
+    @Mock
+    private CaregiverRepository caregiverRepository;
+
+    @Mock
+    private MemberRepository memberRepository;
+
+    @InjectMocks
+    private CaregiverService caregiverService;
+
+    @Nested
+    class 간병인_프로필_생성 {
+
+        @Test
+        void 간병인_프로필을_성공적으로_생성한다() {
+            // given
+            Member member = huseongRoleCaregiver();
+            when(memberRepository.findByUsername(member.getUsername())).thenReturn(Optional.of(member));
+            when(caregiverRepository.save(any(Caregiver.class))).thenAnswer(invocation -> invocation.getArgument(0));
+            CaregiverProfileCreateRequest request = createCaregiverProfileRequest();
+
+            // when
+            CaregiverProfileResponse response = caregiverService.saveCaregiverProfile(member.getUsername(), request);
+
+            // then
+            assertNotNull(response);
+            assertThat(response.getMemberId()).isEqualTo(member.getId());
+            assertThat(response.getName()).isEqualTo(request.getName());
+        }
+
+        @Test
+        void 간병인_프로필을_생성할_때_권한이_없으면_예외가_발생한다() {
+            // given
+            Member member = huseongRolePatient();
+            CaregiverProfileCreateRequest request = createCaregiverProfileRequest();
+            when(memberRepository.findByUsername(member.getUsername())).thenReturn(Optional.of(member));
+
+            // when & then
+            assertThatThrownBy(() -> caregiverService.saveCaregiverProfile(member.getUsername(), request))
+                    .isInstanceOf(AuthorizationException.class);
+        }
+
+        //TODO
+//    @Test
+//    @DisplayName("간병인 프로필을 생성할 때 중복 가입이면 예외가 발생한다.")
+//    void 간병인_프로필을_생성할_때_중복_가입이면_예외가_발생한다() {
+//        // given
+//        Member member = huseongRoleCaregiver();
+//        when(memberRepository.findByUsername(member.getUsername())).thenReturn(Optional.of(member));
+//        when(caregiverRepository.findByMember(member)).thenReturn(Optional.of(caregiver));
+//        CaregiverProfileCreateRequest request = createCaregiverProfileRequest();
+//
+//        // when, then
+//        assertThatThrownBy(() -> caregiverService.saveCaregiverProfile(member.getUsername(), request))
+//                .isInstanceOf(EntityNotFoundException.class);
+//    }
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Comment has incorrect indentation level 0, expected is 8, indentation should be the same level as line 70.